test/functional/Java8andUp/src/org/openj9/test/attachAPI/TestAttachErrorHandling.java

```diff
@@ -96,15 +96,17 @@ public void testTracepoint() {
 	public void test_startupShutdownRobustness() throws IOException {
 		File ipcDir=new File(System.getProperty("java.io.tmpdir"), testName);
 		ipcDir.mkdir();
+		logger.debug("test_startupShutdownRobustness ipcDir = " + ipcDir.getAbsolutePath());
 		File logDir=new File(System.getProperty("java.io.tmpdir"), testName+"_logs");
 		logDir.mkdir();
+		logger.debug("test_startupShutdownRobustness logDir = " + logDir.getAbsolutePath());
 		
 		ArrayList<TargetManager> targets = new ArrayList<TargetManager>();
 		ArrayList<String> vmArgs = new ArrayList<String>();
 		vmArgs.add("-Dcom.ibm.tools.attach.directory="+ ipcDir.getAbsolutePath());
 		vmArgs.add("-Dcom.ibm.tools.attach.logging=yes");
 		vmArgs.add("-Dcom.ibm.tools.attach.log.name="+logDir.getAbsolutePath()+File.separatorChar);
-		for (int i = 0; i < 10; ++i)  {
+		for (int i = 0; i < 5; ++i)  {
```

Comment:
Since only AIX has a problem with the larger number, perhaps this change should be platform specific so as not to change the testing for other platforms.

Reply:
It occurred in `OSX` as well just less often than `AIX`. To ensure all the `targetVM` can be waked, the `numTargets` within `CommonDirectory.notifyVm(numTargets, true)` must not less than the actual `targetVM` number.